streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractSessionBytesStoreTest.java

```diff
@@ -77,7 +77,7 @@
 
     private MockRecordCollector recordCollector;
 
-    private InternalMockProcessorContext context;
```

Comment:
Why was this needed?

Reply:
It looks like @patrickstuedi needed to reference the context in the concrete test classes that subclass this.